Samples/BackgroundBlur/BackgroundBlur/SegmentModel.cpp

```diff
@@ -0,0 +1,285 @@
+#include "SegmentModel.h"
+
+#include <iostream>
+#include <filesystem>
+
+using winrt::Windows::Foundation::PropertyValue;
+using winrt::hstring;
+using namespace winrt;
+using namespace Windows::Foundation::Collections;
+
+enum OnnxDataType : long {
```

Comment:
is this defined in an onnx header somewhere that we can include?

Reply:
I got this enum from the Tensorization sample in the gallery [here](https://github.com/microsoft/Windows-Machine-Learning/blob/user/linneamay/stream-async/Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/TensorizationModels.cs), it looks like it's not defined in any onnx header but @smk2007 can probably confirm.

Reply:
Its part of the onnx protobuf... not easily accessible from c#